src/solvers/refinement/string_constraint_generator_valueof.cpp

```diff
@@ -514,56 +513,60 @@ exprt string_constraint_generatort::add_axioms_for_correct_number_format(
 }
 
 /// add axioms corresponding to the Integer.parseInt java function
-/// \par parameters: function application with one string expression
+/// \param f: a function application with either one string expression or one
+///   string expression and an expression for the radix
 /// \return an integer expression
 exprt string_constraint_generatort::add_axioms_for_parse_int(
   const function_application_exprt &f)
 {
-  string_exprt str=get_string_expr(args(f, 1)[0]);
+  PRECONDITION(f.arguments().size()==1 || f.arguments().size()==2);
+  string_exprt str=get_string_expr(f.arguments()[0]);
+  const exprt radix=
+    f.arguments().size()==1?from_integer(10, f.type()):f.arguments()[1];
```

Comment:
I don't see anything indicating that in the coding standard. I had a quick look in the codebase and some sections use spaces and some don't. @tautschnig seems to be the authority on formatting so I found a PR of mine that he reviewed with a ternary operator in it and he didn't disapprove of having no spaces: https://github.com/diffblue/cbmc/pull/767/files#diff-ffd0a317958e24d1f65f21ed40f67c99R55